.teamcity/settings.kts

```diff
@@ -60,6 +61,24 @@ project {
         param("teamcity.ui.settings.readOnly", "true")
     }
 
+    features {
+        untrustedBuildsSettings {
+            id = "UntrustedBuildsPolicy"
+            defaultAction = UntrustedBuildsSettings.DefaultAction.APPROVE
+            approvalRules = "group:ALL_USERS_GROUP:1"
+            manualRunsApproved = true
+            enableLog = true
+        }
+
+        youtrack {
+            id = "YouTrack"
+            displayName = "JetBrains YouTrack"
+            host = "https://youtrack.jetbrains.com/"
+            projectExtIds = "KTOR"
+            accessToken = "%youtrack.token%"
```

Comment:
Is it possible to generate a non-personal token for YouTrack?

Reply:
nope

Reply:
How can I add a property to CI?